node/src/neighborhood/node_record.rs

```diff
@@ -321,6 +343,9 @@ pub struct NodeRecordMetadata {
     pub last_update: u32,
     pub node_addr_opt: Option<NodeAddr>,
     pub unreachable_hosts: HashSet<String>,
+    pub node_location_opt: Option<NodeLocation>,
+    pub node_distrust_score: u32,
```

Comment:
What is this `node_distrust_score` about? I'm suspecting it to be unused or barely useful. Maybe you should just throw it away?